### DISCUSSION TITLE
Shortcut Runner for Mac OSX Monterey

### DISCUSSION THREAD

**Discussion by suniltaneja** [johnlindquist/kit#676] [Share]
Shortcut Runner for Mac OSX Monterey

[Install Shortcut runner for Mac OSX Monterey](https://scriptkit.com/api/new?name=shortcutrun&url=https://gist.githubusercontent.com/suniltaneja/b5755681d3da2fd2b7eacb44b70777ef/raw/01af76f2c96c54ec31457bd729211c6fc84a53a2/shortcutrun.js)

```
// Menu: Shortcut Run
// Description: Shortcut runner for Mac OSX Monterey
// Author: Sunil Taneja
// Twitter: @suniltaneja

import "@johnlindquist/kit"
const list = await exec(`shortcuts list`)
const list_arr = list.stdout.split("\n")
const shortcut = await arg("Enter Shortcut", list_arr)
await $`/usr/bin/shortcuts run ${shortcut}`
```

**Comment by johnlindquist** [comment]
Awesome, thanks for sharing! I just got a Monterey set up a couple weeks ago. I've heard nothing but good things about shortcuts. I'll have to dig in soon.

**Comment by youngkyo0504** [comment]
Awesome...!  

But I have some trouble. script can't work. 

When I run that command exactly the same way, it works fine🤔 

https://github.com/johnlindquist/kit/assets/78121870/cc471b21-1d2d-4a11-8470-5ca94d5286f4

**Reply by alkene5** [reply]
```js
// try this
await exec(`shortcuts run '${shortcut}' &`)
```
